topology/ANL/ANL ASC/ANLASC.yaml

```diff
@@ -0,0 +1,243 @@
+Disable: false
+GridType: OSG Production Resource
+GroupID: 283
+Resources:
+  ANL-ATLAS-GRIDFTP1:
+    Active: true
+    ContactLists:
+      Administrative Contact:
+        Primary: Doug Benjamin
+      Security Contact:
+        Primary: Doug Benjamin
+      Submitter Contact:
+        Primary: Doug Benjamin
+    Disable: false
+    FQDN: atlasgridftp.hep.anl.gov
+    FQDNAliases:
+    - atlasgridftp1.hep.anl.gov
+    ID: 437
+    Services:
+      GridFtp:
+        Description: GridFtp Storage Element
+        Details:
+          hidden: false
+          uri_override: atlas8.hep.anl.gov:2811
```

Comment:
Any idea what `hidden` and `uri_override` actually mean?

Reply:
- `uri_override` is if the service is available on some non-standard URL.
- `hidden`: I have a vague recollection that this was used to filter out various services, especially for FermiGrid back in the day.  No clue if it's still in use.